package.json

```
@@ -55,7 +55,9 @@
     "docker:stop": "./test/docker/docker-stop.sh",
     "typedocs": "typedoc --customCss ./documentation/typedoc.css",
     "jest": "jest --coverageReporters text-summary --",
-    "lint": "eslint . --cache --ignore-path .gitignore --max-warnings 0",
+    "lint": "npm run lint:ts && npm run lint:markdown",
+    "lint:ts": "eslint . --cache --ignore-path .gitignore --max-warnings 0",
+    "lint:markdown": "npx markdownlint-cli2 \"**/*.md\" \"#node_modules\" \"#LICENSE.md\"",
```

```suggestion
    "lint:markdown": "markdownlint-cli2 \"**/*.md\" \"#node_modules\" \"#LICENSE.md\"",
```
If I'm not mistaken this should work. `npm` adds `node_modules/bin` to the path when running scripts.

Why is the license excluded? Doesn't seem like anything is wrong with it?

Would also add the ignored folders/files to the markdownlint file itself.

---

the license doesn't have a heading which also is a violation of the recommended defaults. Don't know if we can just add a heading or if we then should disable the rules altogether? My thought was not to touch the license file

---

And the `npx` is a derp on my account 😄 it's just what I was using locally to test

---

This is also the `MD041` for the license then? If we have so many cases that violate it we should perhaps just disable that rule instead of ignoring all the files that break it.

---

I think MD041 is a good default normally but github templates and a license file just seem like valid exceptions. The config readme has been changed to pass this rule now.